examples/connect-app/src/main.ts

```diff
@@ -0,0 +1,45 @@
+import { type DialOptions } from '@viamrobotics/rpc/src/dial';
```

Comment:
now that org api keys are in, the app example should use api keys https://github.com/viamrobotics/viam-typescript-sdk/pull/164

Reply:
While org api keys are now supported, we have not made strides to easily access the org api keys, per conversation with @jr22. To get the org api key, we would need to [create the key](https://docs.viam.com/manage/cli/#create-an-organization-api-key) by authenticating with an `accessToken`, which is what we're trying to get in the first place by using the key.

Reply:
tested with an actual api key, and ready to go!